README.md

```diff
@@ -47,6 +47,7 @@ Options:
     -a, --admin         User account should be added to admin group.
     -A, --autologin     User account should automatically login.
     --hidden            User account should be hidden.
+    -d, --productbuild  Creates a distribution-style package for use with startosinstall
```

Comment:
Why not `-d, --distribution`?

Reply:
I'm actually surprised that I wrote it that way, that indeed makes much more sense. Will change it.